src/test/java/com/googleinterns/zoomtube/servlets/AuthenticationServletTest.java

```diff
@@ -0,0 +1,129 @@
+package com.googleinterns.zoomtube.servlets;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+
```

Comment:
Why is there a space here?

Reply:
Updated linter to fix these formatting issues